.github/workflows/ci.yml

```diff
@@ -67,6 +67,6 @@ jobs:
           --durations=20
 
       - name: Upload coverage report
-        uses: codecov/codecov-action@18283e04ce6e62d37312384ff67231eb8fd56d24 # v5
+        uses: codecov/codecov-action@fdcc8476540edceab3de004e990f80d881c6cc00 # v5
```

Comment:
This is the hash for [v5.5.0](https://github.com/codecov/codecov-action/releases/tag/v5.5.0).